lib/graphics/index.js

```diff
@@ -72,24 +72,15 @@ app.get('/graphics/:bundleName*', (req, res, next) => {
 	}
 });
 
-app.get('/graphics/:bundle/components/*', (req, res, next) => {
-	const bundleName = req.params.bundle;
-	const bundle = Bundles.find(bundleName);
-	if (!bundle) {
-		next();
-		return;
-	}
-
-	const resName = req.params[0];
-	const fileLocation = path.join(bundle.dir, 'bower_components', resName);
-
-	// Check if the file exists
-	if (!fs.existsSync(fileLocation)) {
-		next();
-		return;
-	}
+function setupBowerComponents(bundles) {
```

Comment:
Likewise, this breaks the ability to add new graphics during runtime. Personally, this slows down my dev workflow.